frontend/src/pages/Graphing/components/Table.tsx

```diff
@@ -56,8 +58,14 @@ export const MetricTable = ({
 
 	const showMetricFn = series.join('') === ''
 
-	const [sortColumn, setSortColumn] = useState(-1)
-	const [sortAsc, setSortAsc] = useState(true)
+	const [sortAsc, setSortAsc] = useLocalStorage<boolean>(
+		`sort-asc-${visualizationId}`,
```

Comment:
If this is "undefined", will it be set for all not saved visualizations?

Reply:
yep, ideally it would persist after saving, but this is a bit complicated because we're saving the cached graph with a temp id and then the id is replaced after apollo refetches the visualization, but there's not an easy way to link the two in order to replace the local storage key. I figured this approach was ok for now